aboutMe.css

```diff
@@ -0,0 +1,9 @@
+body.font-family: Geramond;
+p.background-color:#BCD2EE;
```

Comment:
These first few elements are not working because the syntax is incorrect. All of them need to follow the structure of your last element for example: 
p{
  background-color: red
}